boards: nucleo_l476rg: Fix arduino_i2c

erwango opened the pull request:
On nucleo_l476rg, arduino I2C is actually i2c1 using PB8/PB9
and not I2C3. Fix this.

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

erwango commented:
@pabigot, can you have a check on your side?

pabigot reviewed (changes requested):
Bus changes appear to be correct, but no devices respond on my board.

erwango committed:
boards: nucleo_l476rg: Fix arduino_i2c

On nucleo_l476rg, arduino I2C is actually i2c1 using PB8/PB9
and not I2C3. Fix this.

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

erwango commented on boards/arm/nucleo_l476rg/nucleo_l476rg.dts:
```diff
@@ -54,11 +54,16 @@
+&i2c3 {
+	status = "ok";
+	clock-frequency = <I2C_BITRATE_FAST>;
+}
```
right, txs

erwango replied:
fixed

pabigot commented:
Before I spend more time on this: I am using CN7.36 as SDA and CN7.38 as SCL; is that what you expect?

It looks like the switch to I2C1 for those pins should have worked, but the i2c scanner app doesn't find a device that is found if I2C3 is selected instead.

I can't swear this isn't a problem on my end, since I rarely use this hardware (only for i2c testing).  How have you tested it?

erwango commented:
I've tested it using x-nucleo-iks01a1 expansion board.
This is also what is described in [nucleo-64 user manual](https://www.st.com/content/ccc/resource/technical/document/user_manual/98/2e/fa/4b/e0/82/43/b7/DM00105823.pdf/files/DM00105823.pdf/jcr:content/translations/en.DM00105823.pdf), page 52. So this is actually CN5.10(I2C 1 SCL) and CN5.9 (I2C 3 SDA)
